javascript/core/flow-control/loops-part-2.md

```diff
@@ -34,84 +19,99 @@ links:
 ---
 ## Content
 
-*for in* loops allow you to iterate through an objects keys. E.g. the keys below are firstName and lastName:
+`for..in` loops allow you to iterate through an objects keys. E.g. the keys below are `firstName` and `lastName`:
 
 ```javascript
-var obj = {
+let obj = {
    firstName: "John",
    lastName: "Smith"
 };
 ```
 
 The keys can be used to get the value:
+
 ```javascript
-for (var i in obj) {
-   console.log(i); //key  
-   console.log(obj[i]); //value
-} //firstName John lastName Smith
+for (let key in obj) {
+   console.log(key, obj[key]);
+}
+
+// "firstName" "John"
+// "lastName" "Smith"
 ```
 
 **For of**
 
-*for of* loops are used with iterable objects such as arrays & maps and allow you to easily loop through an iterables values:
+`for..of` loops are used for looping through iterable objects such as arrays:
 
 ```javascript
-var iterable = [10, 20, 30];
-for (var i of iterable) {
+let arr = [10, 20, 30];
+
+for (let i of arr) {
    console.log(i); //10 20 30
 }
 ```
 
 **Why use for of?**
```

Comment:
Suggestion:
"**Why use for of?**" -> "**Why use `for of`?** This can help non-fluent English users.

Reply:
Same can be applied to the "**For of**" on line `42`

Reply:
```suggestion
**Why use `for..of`?**
```